libs/category/src/models/filters/category-filter-option.ts

```diff
@@ -0,0 +1,7 @@
+import { DaffCategoryFilter } from './category-filter';
+
+/**
+ * A given option of a particular filter, e.g. "Blue" is a DaffCategoryFilterOption
+ * of the {@link DaffCategoryFilter} "Color".
+ */
+export type DaffCategoryFilterOption = DaffCategoryFilter['options'][string];
```

Comment:
Would it be better for this to be an interface that could be extended by `DaffCategoryFilterEqualOption`?

Reply:
Yes and no. There's a new type I added 9246789f but this is nice to use as the union of all possible types.

Reply:
I think a more useful (or maybe less confusing) union of all possible types would be more similar to the `DaffCategoryFilterRequest`. I'm not completely sure if this does that, and I'd suspect it does not. What do you think about this instead:

`export type DaffCategoryFilterOption = DaffCategoryRangeFilterOption | DaffCategoryEqualFilterOption;`

the model names I used aren't correct, but hopefully you get what I'm saying.

Reply:
![Screenshot 2021-04-15 130815](https://user-images.githubusercontent.com/9029654/114910297-b71df000-9deb-11eb-9b2d-bb90c85090e8.png)

Does this do what you expect?

Reply:
Ok sure that seem good